source/user-manual/capabilities/policy-monitoring/rootcheck/how-it-works.rst

```diff
@@ -18,32 +18,42 @@ The *rootcheck* engine can perform the following checks:
-| cis_rhel_linux_rcl.txt   | Based on `CIS Benchmark for Red Hat Enterprise Linux v1.0.5 <https://benchmarks.cisecurity.org/tools2/linux/CIS_RHLinux_Benchmark_v1.0.5.pdf>`_                                |
-+--------------------------+--------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-| cis_sles11_linux_rcl.txt | Based on `CIS Benchmark for SUSE Linux Enterprise Server 11 v1.1.0 <https://benchmarks.cisecurity.org/tools2/linux/CIS_SUSE_Linux_Enterprise_Server_11_Benchmark_v1.1.0.pdf>`_ |
-+--------------------------+--------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-| cis_sles12_linux_rcl.txt | Based on `CIS Benchmark for SUSE Linux Enterprise Server 12 v1.0.0 <https://benchmarks.cisecurity.org/tools2/linux/CIS_SUSE_Linux_Enterprise_Server_12_Benchmark_v1.0.0.pdf>`_ |
-+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| system_audit_rcl.txt     | Web vulnerabilities and exploits                                                                                                                                               |
-+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| win_audit_rcl.txt        | Check registry values                                                                                                                                                          |
-+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| system_audit_ssh.txt     | SSH Hardening                                                                                                                                                                  |
-+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| win_applications_rcl.txt | Check if malicious applications are installed                                                                                                                                  |
-+--------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+- cis_debian_linux_rcl.txt
```

Comment:
Keep the old table without links. We've recently updated branch 3.8, copy from there